apps/nowcasting-app/components/utils.ts

```diff
@@ -25,18 +25,34 @@ export const getTimeFromDate = (date: Date) => {
   return date.toTimeString().split(" ")[0].slice(0, -3);
 };
 export const formatISODateString = (date: string) => {
+  // Changes the ISO date string to Europe London time, and remove timezone and seconds
+
   const dateid = date?.slice(0, 16);
-  return dateid;
+
+  const d = new Date(date);
+  const date_london_date_str = d.toLocaleDateString("en-GB", { timeZone: "Europe/London" });
+  // make string in DD/MM/YYYY format
+  const date_london_time_str = d
+    .toLocaleTimeString("en-GB", { timeZone: "Europe/London" })
+    .slice(0, 5);
+  // make string in HH:MM format
+  const year = date_london_date_str.slice(6, 10);
+  const month = date_london_date_str.slice(3, 5);
```

Comment:
this function get used to convert all the dates form the api , so when i ran it locally i noticed that clicking on the chart causes the page to freeze . 
this function need to be fast .
i'm looking for a way to convert it without using javascript Date class